### PR TITLE
bump up dependencies

### PR DESCRIPTION
**Pull request by nyuichi**
T/O

### CODE REVIEW

**Comment by nyuichi** [comment]
This will be our next minor version i.e. satysfi-base 1.4.0.
I remember that in the last release it was pointed out that the dependencies were too old. (https://github.com/na4zagin3/satyrographos-repo/commit/59d18ae2da279dcc1316d00ef927b88142fab524)

@na4zagin3 Could you please check if this fixes the issue? I will make a release and send a PR to satyrographos-repo once I get your approval.

**Review by na4zagin3 — APPROVED** [review]
Thanks!  It looks good to me.

Just a nit pick, since now Satyrographos 0.0.2.6 or later is required, we can use GNU-style long options.  I just committed the change.

**Comment by nyuichi** [comment]
> Just a nit pick, since now Satyrographos 0.0.2.6 or later is required, we can use GNU-style long options. I just committed the change.

wow that helps a lot! thank you so much. I'm merging.

**Pull request merged**
